### PR TITLE
Add cSpell for local spell checking

### PR DESCRIPTION
**Pull request by mark-wiemer**
- Only adds it locally, not to CI
- Adds installation docs, requires Node.js
- Currently there are 3,000+ issues in the reader-facing "content" folder, working on fixes now

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "editor.formatOnSave": false,
+  "cSpell.words": [
+    "minetest"
```
Comment (JosiahWI) [inline .vscode/settings.json:4]: Should this be "luanti"? We can also [forbid the word "minetest"](https://cspell.org/docs/forbidden-words/). That could help us catch mistakes, but I can see the word being used intentionally in a page specifically about historical context. Do you think it's worth having to put an ignore comment in those places?

**Comment by mark-wiemer** [comment]
Marking as draft, current npm script doesn't respect workspace dict, my bad

File: .vscode/settings.json (hunk 1 above)
Comment (mark-wiemer) [reply .vscode/settings.json:4]: For now the word `minetest` is still a very valid word on this site, esp. as a lot of code snippets may still reference it, and a lot of historical references will be in place. We can look at forbidding minetest separately, I think.

**Comment by mark-wiemer** [comment]
Closing in favor of #88

**Pull request closed without merging**